### PR TITLE
release 0.6

### PR DESCRIPTION
**Pull request by jordens**
- ad9959: add metadata
- add ad9959 version

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,10 @@
 name = "stabilizer"
 version = "0.5.0"
```
Comment (ryan-summers) [inline Cargo.toml:3]: Need to update this version as well

**Review by ryan-summers — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (jordens) [reply Cargo.toml:3]: Yes. I'll be using `cargo-release` for this which will do the increment in the end just before committing that and publishing.
Next time i'll look into splitting the sequence.

Comment (jordens) [reply Cargo.toml]: In fact I'll try that workflow now.

**Review by ryan-summers — APPROVED** [review]

**Pull request merged**